python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -331,18 +393,20 @@ def defectIngest(repo, calib_repo, defectfiles):
     - They will be manually uncompressed and saved in calib_repo/defects/<tarballname>/.
     - They will be added to the calib registry, but not linked like the flats and biases
     '''
+    lsst.log.configure()
```

Comment:
You should only [need to call this once](https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/log.html#initialization).